src/System.Net.Requests/pkg/System.Net.Requests.pkgproj

```diff
@@ -2,14 +2,8 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\3.9.0\System.Net.Requests.depproj">
-      <SupportedFramework>wp8</SupportedFramework>
-    </ProjectReference>
-    <ProjectReference Include="..\ref\4.0.0\System.Net.Requests.depproj">
-      <SupportedFramework>net45;netcore45;wpa81</SupportedFramework>
-    </ProjectReference>
     <ProjectReference Include="..\ref\System.Net.Requests.csproj">
-      <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
These are the new dependencies being added to the revised 4.1 System.Net.Requests contract:

```
"System.Collections.Specialized": "4.0.0",
"System.ComponentModel": "4.0.0",
"System.ComponentModel.EventBasedAsync": "4.0.0",
"System.Text.Encoding": "4.0.0",
```

Is it because of any of these which drags in netstandard17?  I'm trying to understand exactly which one is doing it.